docs/pages/get-started/installation.md

```diff
@@ -4,50 +4,64 @@ title: Installation
+
 - [Node.js LTS release](https://nodejs.org/en/)
 - [Git](https://git-scm.com)
-- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) for macOS or Linux users
+- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) required only for macOS or Linux users
 
-> Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases."
+> Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS, or Maintenance LTS releases."
 
-### Recommended Tools
+### Recommended tools
 
 - [VS Code Editor](https://code.visualstudio.com/download)
-  - [VS Code Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for **app.json** debugging and autocomplete.
+  - [VS Code Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for **app.json** debugging and autocomplete
 - [Yarn](https://classic.yarnpkg.com/en/docs/install)
-- Windows users: [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows), Bash via WSL, or the VS Code terminal.
+- Windows users: [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows), Bash via WSL, or the VS Code terminal
 
 ### Installing Expo CLI
 
+Expo CLI is installed as a global npm package from a terminal app on your development machine. You will use it for different tasks in the development life-cycle of your project.
+
 <Terminal cmd={[
   '# Install the command line tools',
   '$ npm install --global expo-cli'
 ]} />
 
-Verify that the installation was successful by running `expo whoami`. You're not logged in yet, so you will see "Not logged in". You can create an account by running `expo register` if you like, or if you have one already run `expo login`, but you also don't need an account to get started.
+To verify the successful installation of CLI, run the following command:
+
+<Terminal cmd={['$ expo whoami']} />
+
+If the installation is successful, you will see a "Not logged in" message since you are not logged in to an Expo account yet. You do not need an account to start and can proceed further with your project. However, if you want to register a new expo account, run the command:
+
+<Terminal cmd={['$ expo register']} />
+
+If you already have an Expo account, you can log into it by running the command:
+
+<Terminal cmd={['$ expo login']} />
 
-> 😳 **Need help?** Try searching the [forums](https://forums.expo.dev) &mdash; which are a great resource for troubleshooting.
+> **Need help?** Try searching the [forums](https://forums.expo.dev) &mdash; which is a great resource for troubleshooting.
```

Comment:
Why 'is'?

Reply:
Forums is a single place/website. To be grammatical correct, it can either be `... are great resources for troubleshooting.` or `which is a great resource for troubleshooting.`. LMK if you think the former sounds more grammatically accurate. I will update this line.

Reply:
Yeah, I'd prefer the first, 'forums' is plural, regardless of whether it indicates a single place (but I'd agree on the edge case aspect of it)

Reply:
Reverted in new commits. Kept the plural form.